src/pe_source/data/pe_db/config.py

```diff
@@ -14,40 +15,32 @@
 REPORT_DB_CONFIG = files("pe_reports").joinpath("data/database.ini")
 
 
-# Setup logging to central file
-# To avoid a circular reference error which occurs when calling app.config["LOGGER"]
-# we are directly calling the logger here
-LOGGER = logging.getLogger(__name__)
```

Comment:
I think removing this logger statement will cause the Logs to no longer go to the centralized logging file. Ignore me if I am wrong